src/stlink-lib/usb.h

```diff
@@ -73,8 +70,5 @@ stlink_t *stlink_open_usb(enum ugly_loglevel verbose, enum connect_type connect,
 size_t stlink_probe_usb(stlink_t **stdevs[], enum connect_type connect, int freq);
 void stlink_probe_usb_free(stlink_t **stdevs[], size_t size);
 
-#ifdef __cplusplus
-}
-#endif
 
```

Comment:
```suggestion
```

Reply:
Addressed in commit da3d9e3f1e1a6d116df5830cf506f30ff56196b3.